lib/EResourceType/tests/interactor.js

```diff
@@ -4,5 +4,5 @@ import {
 } from '@bigtest/interactor';
 
 export default @interactor class EresourceTypeInteractor {
-  type = text('[data-test-eresource-type]');
+  publicationType = text('[data-test-eresource-type]');
```

Comment:
could we change the `data-test-*` attributes name from type to publicationType just to stay consistent. You would have to reflect this change at the app level as well